lib/associations/belongs-to-many.js

```diff
@@ -464,6 +465,10 @@ BelongsToMany.prototype.injectGetter = function(obj) {
       }
     }
 
+    if (options.hasOwnProperty('schema')) {
+      model = model.schema(options.schema, options.schemaDelimiter);
+    }
```

Comment:
Could the user not just pass `Model.schema()` to the relation setup call instead?